package/CHANGELOG

```diff
@@ -13,6 +13,9 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
+08/30/22 rafaelpap
```

Comment:
Thanks for the fix @rafaelpap. The CHANGELOG entries need to go underneath the version number (`2.3.0`) and are ordered newest first.   A good place to put this would be under `Fixes` :)

Reply:
Hold for 2.4.0